src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -592,7 +592,7 @@ bool MetaspaceShared::link_class_for_cds(InstanceKlass* ik, TRAPS) {
   return res;
 }
 
-void MetaspaceShared::link_and_cleanup_shared_classes(TRAPS) {
+void MetaspaceShared::link_shared_classes(TRAPS) {
   // Collect all loaded ClassLoaderData.
   ResourceMark rm;
```

Comment:
(This is pre-existing) I think the ResourceMark can be removed.

Reply:
The function is called by both static and dynamic dump, for dynamic caller already set ResourceMark, but static. I will leave this as it is, it may need more check to see if it should be removed. Maybe it can be removed since I did not see any use of the it in the call and its callees.